### ISSUE TITLE
Update to Java 17

### ISSUE THREAD

**Issue by cerus** [cerus/deploy-helper#10]
Update the project to Java <del>16</del> 17. Make sure to test everything and to adjust the README.

Comment if you would like to work on this.

**Comment by greylurk** [comment]
Is there a reason not to move to Java 17?

**Comment by cerus** [comment]
> Is there a reason not to move to Java 17?

Is Java 17 already released?

**Comment by greylurk** [comment]
Yes, as of 9/15/2021.  And it's an LTS, which theoretically means it'll be around for a while.

**Comment by cerus** [comment]
Wow, how did I not notice the release? I'm gonna edit this issue, thank you for pointing that out!
@greylurk would you like to work on this issue?

**Comment by greylurk** [comment]
Sure.  I'm not 100% clear on how to test this, but I'll take a swing at it.

**Comment by cerus** [comment]
Just make sure that the program compiles and runs without any errors :)

Edit - Please also make sure to read the [contribution guidelines](https://github.com/cerus/deploy-helper/blob/c63ec918ab255dabceae3d7e7aac96a6b40bfb31/CONTRIBUTING.md). Thanks! (You can ignore the first rule because this issue already exists)
